src/main/webapp/app/index.module.scss

```diff
@@ -22,6 +22,14 @@
   font-size: $bigger-default-font-size;
 }
 
+:global(.btn).actionButton {
```

Comment:
I think we've been using `-` in class names

Reply:
This is module class name, in this case, we mostly use camelCase.